boilerplate/skeleton/extension/js/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@<%= params.packageName %>",
   "private": true,
+  "version": "0.0.0",
```

Comment:
Actually, just one quick thing. Please add the "version" key to the "core" module in the scaffolder generation function:

https://github.com/flarum/cli/blob/97a27835eb8ab8b919591cc87c373d08616d5c13/src/steps/gen-ext-scaffolder.ts#L387-L388